src/components/TodoList/TodoList.tsx

```diff
@@ -1,224 +1,92 @@
 /* eslint-disable */
 import React from 'react';
+import { Todo } from '../../types/Todo';
+import { useAppSelector } from '../../app/store';
+import { filterSelector } from '../../features/filter';
+import classNames from 'classnames';
+import { useDispatch } from 'react-redux';
+import { actions, currentTodoSelector } from '../../features/currentTodo';
+
+type Props = {
+  todos: Todo[];
+};
 
-export const TodoList: React.FC = () => {
-  return (
-    <>
+export const TodoList: React.FC<Props> = ({ todos }) => {
+  const filter = useAppSelector(filterSelector);
+  const currentTodo = useAppSelector(currentTodoSelector);
+  const dispatch = useDispatch();
+
+  if (!todos.length && !!filter.query) {
```

Comment:
The condition `!!filter.query` assumes that `filter.query` is a string or a falsy value. Ensure that `filter.query` is correctly typed and initialized to avoid unexpected behavior.